.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libs/providers/flagd/schemas"]
+	path = libs/providers/flagd/schemas
+	url = https://github.com/james-milligan/schemas.git
```

Comment:
Update to the real schema repo when you're ready to merge.

Reply:
awaiting changes in https://github.com/open-feature/schemas/pull/13